consensus/clique/src/main/java/tech/pegasys/pantheon/consensus/clique/CliqueVotingBlockInterface.java

```diff
@@ -30,34 +30,34 @@
   public static final Address NO_VOTE_SUBJECT =
       Address.wrap(BytesValue.wrap(new byte[Address.SIZE]));
 
-  private static final ImmutableBiMap<ValidatorVotePolarity, Long> voteToValue =
+  public static final ImmutableBiMap<VoteType, Long> voteToValue =
```

Comment:
This probably should stay `private`.  See comments below for suggestions on how to make that work.